atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -486,15 +486,6 @@ void runOneTimeStartupChecks() {
         }
     }
 
-    //todo dedupe this into a name-demangling class that everyone can access
-    protected static String internalTableName(TableReference tableRef) {
-        String tableName = tableRef.getQualifiedName();
-        if (tableName.startsWith("_")) {
-            return tableName;
-        }
-        return tableName.replaceFirst("\\.", "__");
-    }
-
```

Comment:
Drive-by deletion, as this didn't seem to be used anywhere, and this exact method is available on `AbstractKVS.internalTableName`